src/ergw_aaa_diameter_srv.erl

```diff
@@ -156,6 +156,7 @@ handle_plain_error([Module | Extra], Error, Request, SvcName, CallOpts)
 handle_plain_error(Error, _Request, _SvcName, _App, _CallOpts, []) ->
     {error, Error};
 handle_plain_error(Error, Request, SvcName, App, CallOpts, [SI|T]) ->
+    ?LOG(error, "ACR failed in ~p with: ~p", [hd(proplists:get_value(module, SI)), Error]),
```

Comment:
Will proplists:get_value(module, SI) always return a list?
It would be unfortunate if this crashed because we want to log something.

Also, proplists:get_value(module, SI) will be done later (line 162). Might be worth saving the value for then.

Reply:
it's hard to say if it's always a list, but if there are such cases, I added a default value